src/pages/patterns/drag-and-drop.mdx

```diff
@@ -0,0 +1,596 @@
+2. Use a **horizontal indicator** to communicate placement between items in a column direction list (top to bottom)
+
+### Dropzones
+
+Dropzones are locations on the screen where the item can be moved onto. For flat structure lists,
+the dropzone is usually the list container. In complex workflows, dropzone might be another list — flat or nested,
+or even another item. Dropzones can be communicated in several ways, by using dashed borders, labels or a
+task-oriented context of reordering or building something on the screen.
+
+#### Border treatment
+
+![Three examples showing a dropzone in the default, active, and highlighted state.](figma:patterns-dnd-formatting-dropzone-states.png)
+
+1. Use `1px grey-600` border to show empty **default dropzones**
+2. Use a dashed `1px blue-600` border to show **active dropzones** where grabbed items can be placed
+3. Use solid `2px blue-600` border to communicate a **highlighted dropzone**
+
+#### Text and icons
+
+Dropzones, like good empty states, should explain how to interact with it. Sometimes, the content in
+the dropzone will appear after the user drags the item so only “drop” is mentioned in the content.
+Include the consequence if it’s important for the user to understand what will happen when they drop
+the item in this dropzone. Content, together with supporting icons and colors will make the user
+confident in the end result.
+
+![Three different examples showing a default dropzone, a dropzone with a danger theme applied,
+and a highlighted dropzone.](figma:patterns-dnd-formatting-dropzone-examples.png)
+
+1. **Default dropzone** without icon
+2. **Danger dropzone** with a theme and icon applied
+3. **Highlighted dropzone** with a custom icon
+
+Use icons to reinforce the labels when needed. Hide decorative icons from assistive technologies.
+
+#### Empty and populated dropzones
+
+Use borders and labels to make empty dropzones discoverable. Keep populated dropzones simple by showing only items inside.
+Always provide the label for screen readers, even if it's not visible on the screen.
+
+![An empty dropzone on the left and a group with several draggable items.](figma:patterns-dnd-formatting-dropzone-empty-populated.png)
+
+1. Include content that will explain how to interact with the empty dropzone
+2. Populated dropzones don’t need a supporting call-to-action
+
+#### Dropzones as items
+
+Provide visual affordance when items are both draggable and potential dropzones.
+
+![Example of moving a draggable item and replacing another item with it.](figma:patterns-dnd-formatting-dropzone-replace.png)
+
+1. Item with default styling
+2. Item as an active dropzone with a dashed border
+3. Item as a highlighted dropzone
+
+## Flows
+
+### Reordering a list
+
+This is the most basic example of drag-and-drop. Interaction is discoverable through a grip handle on the item.
+See [Draggable tables](https://garden.zendesk.com/components/table#draggable) for reordering rows.
```

Comment:
```suggestion
See [draggable tables](https://garden.zendesk.com/components/table#draggable) for reordering rows.
```

[nit] should we lowercase since this isn't referring directly to a component, but rather an example?

Reply:
Yes. I think lowercase is better.